kibbeh/src/ui/RoomPanelIconBar.tsx

```diff
@@ -35,7 +35,12 @@ export const RoomPanelIconBar: React.FC<RoomPanelIconBarProps> = ({
     <div className="justify-between bg-primary-700 rounded-b-8 py-3 px-4 w-full">
       <div>
         {mute ? (
-          <BoxedIcon transition className="mr-2" color="800" onClick={() => mute.onMute()}>
+          <BoxedIcon
+            transition
+            className="mr-2"
+            color="800"
+            onClick={() => mute.onMute()}
```

Comment:
```suggestion
            onClick={mute.onMute}
```

Reply:
I'm pretty sure the reason I did that was because TypeScript was screaming at me.